openbb_platform/core/openbb_core/app/router.py

```diff
@@ -343,35 +360,41 @@ def complete_signature(
     def inject_return_type(
         func: Callable[P, OBBject], inner_type: Any, outer_type: Any
     ) -> Callable[P, OBBject]:
-        """Inject full return model into the function.
-        Also updates __name__ and __doc__ for API schemas."""
+        """
+        Inject full return model into the function.
+
+        Also updates __name__ and __doc__ for API schemas.
+        """
         ReturnModel = inner_type
-        if get_origin(outer_type) == list:
+        outer_type_origin = get_origin(outer_type)
+
+        if outer_type_origin == list:
             ReturnModel = List[inner_type]  # type: ignore
-        elif get_origin(outer_type) == Union:
+        elif outer_type_origin == Union:
             ReturnModel = Union[List[inner_type], inner_type]  # type: ignore
 
         return_type = OBBject[ReturnModel]  # type: ignore
         return_type.__name__ = f"OBBject[{inner_type.__name__}]"
         return_type.__doc__ = f"OBBject with results of type '{inner_type.__name__}'."
-        return_type.model_rebuild(force=True)
```

Comment:
@tehcoderer do you see any problem with removing this line? I don't think it's doing anything either on the python interface and the API